src/main/java/net/sf/jabref/logic/undo/UndoRedoEvent.java

```diff
@@ -0,0 +1,10 @@
+package net.sf.jabref.logic.undo;
+
+
+public class UndoRedoEvent extends AddUndoEvent {
```

Comment:
Basically to avoid copying code as they have exactly the same content but
two different types are needed. An alternative might be to add the type in
the event but that seemed worse.